packages/kkmarkdown/kkmarkdown.1.0.0/opam

```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
+name: "kkmarkdown"
+version: "1.0.0"
+synopsis: "A safe markdown engine"
+description: """
+The goals of kkmarkdown are,
+
+* To avoid XSS attack, by supporting limited markdown syntax.
+* To run the same markdown engine in both server and client sides, thanks to js_of_ocaml.
+"""
+maintainer: "kkeundotnet <kkeundotnet@gmail.com>"
+authors: "kkeundotnet <kkeundotnet@gmail.com>"
+license: "Unlicense"
+homepage: "https://github.com/kkeundotnet/kkmarkdown"
+bug-reports: "https://github.com/kkeundotnet/kkmarkdown/issues"
+dev-repo: "git+https://github.com/kkeundotnet/kkmarkdown.git"
+depends: [
+  "dune" {build}
```

Comment:
```suggestion
  "dune" {>= "3.6"}
```
According to the dune-project file. Note that due to a bug in dune (https://github.com/ocaml/dune/issues/2147), it should never be tagged as a build dependency.

Reply:
Thank you. Good to know that.